### ISSUE TITLE
Crash in v3

### ISSUE THREAD

**Issue by bendiktv2** [looker-open-source/look-at-me-sideways#134]
Hello, when running `lams --reporting=yes` in a certain looker-codebase, lams crashes with the following output:

```
Parsing project...
TypeError: (model[type][("+" + name)] || []).map is not a function
    at recurse (/Users/me/.nvm/versions/node/v14.15.0/lib/node_modules/@looker/look-at-me-sideways/node_modules/lookml-parser/lib/transformations/apply-extensions-refinements.js:128:34)
    at findObjectsToApply (/Users/me/.nvm/versions/node/v14.15.0/lib/node_modules/@looker/look-at-me-sideways/node_modules/lookml-parser/lib/transformations/apply-extensions-refinements.js:111:18)
    at Object.transformations_applyExtensionsRefinements [as applyExtensionsRefinements] (/Users/me/.nvm/versions/node/v14.15.0/lib/node_modules/@looker/look-at-me-sideways/node_modules/lookml-parser/lib/transformations/apply-extensions-refinements.js:42:35)
    at Object.lookmlParser_parseFiles [as parseFiles] (/Users/me/.nvm/versions/node/v14.15.0/lib/node_modules/@looker/look-at-me-sideways/node_modules/lookml-parser/lib/parse-files/index.js:99:9)
    at async module.exports (/Users/me/.nvm/versions/node/v14.15.0/lib/node_modules/@looker/look-at-me-sideways/index.js:67:25)
    at async /Users/me/.nvm/versions/node/v14.15.0/lib/node_modules/@looker/look-at-me-sideways/cli.js:28:2
```

I have not been able to create a minimal-reproducible example for you, but I **think** I have narrowed it down to a view with 2 different refinements (autogenerated view -> refined view -> another refined view)

**Comment by fabio-looker** [comment]
Thanks for the report. I think I should be able to address this even without a repro.

From the message, I am wondering if there might be a refinement that does not match any base view?

**Comment by bendiktv2** [comment]
I got a simple Minimal Reproducible Example here: https://github.com/bendiktv2/lams-key-bug/tree/double-refinement-parse-error (ensure you are on the `double-refinement-parse-error` branch)

**Comment by fabio-looker** [comment]
Hi @bendiktv2 

This should be resolved in the 3.0.0 release

General v3 release notes are here: Updated in [v3](https://looker-open-source.github.io/look-at-me-sideways/release-notes/v3)

Thanks again for the report. Feel free to comment if you still see any issues and I'll re-open
